include/swift/ClangImporter/ClangImporter.h

```diff
@@ -78,14 +80,20 @@ class ClangImporter final : public ClangModuleLoader {
   /// \param ctx The ASTContext into which the module will be imported.
   /// The ASTContext's SearchPathOptions will be used for the Clang importer.
   ///
-  /// \param clangImporterOpts The options to use for the Clang importer.
+  /// \param importerOpts The options to use for the Clang importer.
+  ///
+  /// \param swiftPCHHash A hash of Swift's various options in a compiler
+  /// invocation, used to create a unique Bridging PCH if requested.
+  /// \see -pch-output-dir
```

Comment:
Nitpick: This won't be attached to the `\param` declaration, so it might be better to just make it prose.